build-tools/scripts/XABuildConfig.cs.in

```diff
@@ -15,6 +15,7 @@ namespace Xamarin.Android.Tools
 			{        "arm", @NDK_ARMEABI_V7_API@},
 			{  "arm64-v8a", @NDK_ARM64_V8A_API@},
 			{      "arm64", @NDK_ARM64_V8A_API@},
+			{    "aarch64", @NDK_ARM64_V8A_API@},
```

Comment:
What's `aarch64`, how's it differ from `arm64-v8a`, and why do we need it?

Reply:
We use different prefixes than NDK tools for the same architecture. There are *3* incarnations for arm64

Reply:
```
$ rgrep aarch64 *
Tasks/NdkUtils.cs:					targetPrefix = "--target=aarch64-linux-android";
Tasks/NdkUtils.cs:			// architecture and API level. For instance, the clang/clang++ compilers targetting aarch64 on API level
Tasks/NdkUtils.cs:			//   aarch64-linux-android28-clang
Tasks/NdkUtils.cs:			//   aarch64-linux-android28-clang++
Tasks/NdkUtils.cs:			//   aarch64-linux-android28-clang.cmd
Tasks/NdkUtils.cs:			// `aarch64-linux-android28-clang++.cmd` and so invocation of `clang.exe` becomes harder and,
Tasks/NdkUtils.cs:					return "aarch64-linux-android";
Tasks/NdkUtils.cs:				return "aarch64-linux-android";
Tasks/LinkApplicationDSOs.cs:					case "aarch64":
Tasks/LinkApplicationDSOs.cs:						elf_arch = "aarch64linux";
Tasks/MakeBundleNativeCodeExternal.cs:				case "aarch64":
Tasks/NdkUtilsOld.cs:					archDir = "aarch64-linux-android";
Tasks/NdkUtilsOld.cs:				return Directory.GetDirectories (toolchainsPath, "aarch64-linux-android-*");
Tasks/NdkUtilsOld.cs:				return "aarch64-linux-android-";
Tasks/Aot.cs:				case "aarch64":
Tasks/Aot.cs:					mtriple = "aarch64-linux-android";
Tasks/CompileNativeAssembly.cs:					case "aarch64":
Tasks/CompileNativeAssembly.cs:						prefix = Path.Combine (sdkBinDirectory, "aarch64-linux-android");
Tests/Xamarin.Android.Build.Tests/Utilities/EnvironmentHelper.cs:					case "aarch64":
Xamarin.Android.Build.Tasks.targets:      <_ToolchainPrefix Include="aarch64-linux-android">
Xamarin.Android.Build.Tasks.targets:        <ToolPrefix>aarch64-linux-android</ToolPrefix>
```

Reply:
This needs to be tidied up, but not in this PR.

Reply:
https://twitter.com/infinite_scream